jarbas/chamber_of_deputies/fields.py

```diff
@@ -15,11 +13,10 @@ def deserialize(cls, value, *args, **kwargs):
 
 
 class DateAsStringField(fields.DateField):
-    INPUT_FORMAT = '%Y-%m-%dT%H:%M:%S'
+    INPUT_FORMAT = '%Y-%m-%d %H:%M:%S'
```

Comment:
We may need to review this format again once toolbox gets updated.

Reply:
As this is a date (not a date & time) we could shorten it to `YYYY-MM-DD` (in the toolbox), right?